bin/main.rb

```diff
@@ -0,0 +1,59 @@
+# Set up game
+puts "Welcome!"
+puts "Player 1, What is your name?"
+@player1 = gets.chomp
+puts "Now, Player 2, What is your name?"
+@player2 = gets.chomp
+puts "Let's play! #{@player1} #{@player2}"
+continue_game = true
+
+# While nobody has won
+while continue_game
+  # get input from player 1
+  puts "Choose a number between 1-9 #{@player1}"
+  @player1_input = gets.chomp.to_i
+  # Make sure user has the correct input
+  if @player1_input.is_a?(Integer) && @player1_input > 0 && @player1_input <= 9
+    print "Player one, you have input #{@player1_input}"
+    puts
+    puts "Have you won? press 'y' for yes and 'n' for no 't' for tie"
+    ans = gets.chomp
+    if ans == 'y'
+      continue_game = false
+      break
+    elsif ans == 't'
+      puts 'IT IS A TIE'
+      break
+    end
+  else
```

Comment:
As it's a mock, Instead of asking to the user, I suggest you randomly select from an array the result after from an array
```ruby
[:WIN, :TIE].sample
```

Why? Because in the next milestone, 

1. It's not to the user to decide the outcome of a move
2. At the next milestone, you'll just change that line by calling a function

Reply:
Again, this was to show that there is a flow of whether there is a winner or loser. Since you can't technically "randomize a winner/loser" in the actual game, we figured that we would use this method to allow the user to confirm the winner/loser.

This works the same way and I feel we shouldn't be marked for this.